README.md

```diff
@@ -1,66 +1,88 @@
+composer require bringyourownideas/silverstripe-maintenance
+```
 
-### Installation
-
-Run the following commands to install the package including all suggestions and populate the information initially:
+Build schema and populate database initially:
+```
+vendor/bin/sake dev/build
+```
 
+Optionally install the additional modules:
 ```
-# install the packages
-composer require bringyourownideas/silverstripe-maintenance
-composer require bringyourownideas/silverstripe-composer-security-checker
 composer require bringyourownideas/silverstripe-composer-update-checker
-composer require bringyourownideas/silverstripe-composer-versions
+composer require bringyourownideas/silverstripe-composer-security-checker
+```
+ 
+Build after installing:
+```
+vendor/bin/sake dev/build
+```
+ 
+Run the update task to gather update information if the update-checker module is installed:
+```
+vendor/bin/sake dev/tasks/UpdatePackageInfoTask
+```
+ 
+Run the security task if that module is installed:
+```
+vendor/bin/sake dev/tasks/SecurityAlertCheckTask
+```
 
-# schedule the population of the data
-php ./framework/cli-script.php dev/build
+### Suggested Module Installation
+Note that while the installation of the following modules is optional, it is suggested:
+- [bringyourownideas/silverstripe-composer-security-checker](https://github.com/bringyourownideas/silverstripe-composer-security-checker) checks for known security vulnerabilities
+- [bringyourownideas/silverstripe-composer-update-checker](https://github.com/bringyourownideas/silverstripe-composer-update-checker) checks for available updates of dependencies
+        
 
-# run the queuedjobs
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
-```
+## Source of the Information
 
-*If you don't want to install all packages adjust the command above.*
+The information is based on your composer files. These need to be available in the environment the module is used in. 
+If installed, the modules below process the content of the composer files and check suitable sources for information 
+regarding your set up.
 
+The main functionality comes from these modules:
 
-## Usage
+* [SilverStripe Composer Security Checker](https://github.com/bringyourownideas/silverstripe-composer-security-checker "Check your SilverStripe application for security issues")
+* [SilverStripe Composer Update Checker](https://github.com/bringyourownideas/silverstripe-composer-update-checker "Check your SilverStripe application for available updates of dependencies.")
 
-In the admin section of your SilverStripe website you should see a Maintenance section now. Click on this to view the available information. *You are required to have admin access to view this information.*
+## Documentation
 
+Please see the [user guide](https://github.com/bringyourownideas/silverstripe-maintenance/docs/en/userguide) section.
```

Comment:
I think this should be a relative link - `docs/en/userguide` - that way you won't need to worry about versioning it (master/1/2) (versioning is missing from this URL)